Sources/Load.php

```diff
@@ -849,6 +850,71 @@ function loadUserSettings()
 	call_integration_hook('integrate_user_info');
 }
 
+function loadMinUserSettings($user_id = 0)
+{
+	global $smcFunc, $cache_enable, $modSettings, $language;
+
+	$user_settings_min = array();
+
+	if (empty($userId) || !is_int($userId))
+		return [];
+
+	if (empty($cache_enable) || ($user_settings_min = cache_get_data('user_settings_min-' . $user_id, 120)) == null)
+	{
+		$request = $smcFunc['db_query']('', '
+				SELECT time_offset, additional_groups, id_group, id_post_group, lngfile, smiley_set, time_offset
+				FROM {db_prefix}members
```

Comment:
Would like to see that we use here the mem allias

Reply:
Alias are used to distinct between two or more table columns, this query uses one.

Reply:
would things easier to find and
it doesn't hurt you.